Data-Collection-Inventory/2021Q1_csvs/grant.csv

```diff
@@ -1,32 +1,20 @@
-RF1 Chung grant,"Highly specific, renewable, and cost-effective antibody toolbox for 3D proteomic phenotyping of the brain",1RF1MH121270-01,National Institute of Mental Health,Massachusetts Institute of Technology,9852828
 RF1 Mueller grant,"Accessible technologies for high-throughput, whole-brain reconstructions of molecularly characterized mammalian neurons",1RF1MH121539-01,National Institute of Mental Health,Johns Hopkins University,9868000
 RF1 Nowakowski grant,Mapping Developmental Lineage Relationships in the Cerebral Cortex,1RF1MH121268-01,National Institute of Mental Health,University of California at San Francisco,9852810
 RF1 Regev grant,Scaling up spatial RNA profiling with compressed sensing,1RF1MH121289-01,National Institute of Mental Health,Broad Institute,9852697
 RF1 Tasic grant,Cell class- or type-specific viruses for brain-wide labeling and neural circuit examination,1RF1MH121274-01,National Institute of Mental Health,Allen Institute for Brain Science,9852868
 RF1 Tilgner grant,Single Cell Isoform Expression Across Mouse Brain Regions and Development,1RF1MH121267-01,National Institute of Mental Health,Weill Medical College of Cornell University,9852789
-SFARI_EC grant,No title,SFARI-07-08s,Simons Foundation,University of California at San Diego,
-U01 Chung grant,Towards Integrated 3D Reconstruction of Whole Human Brains at Subcellular Resolution,1U01MH117072-01,National Institute of Mental Health,Massachusetts Institute of Technology,9584926
 U01 Dong grant,Anatomical Characterization of Neuronal Cell Types of the Mouse Brain,1U01MH114829-01,National Institute of Mental Health,University of Southern California,9416006
 U01 Ecker grant,Epigenomic cell-type classification and regulatory element identification in the human brain,1U01MH121282-01,National Institute of Mental Health,Salk Institute for Biological Studies,9852895
 U01 Feng grant,A Molecular and Cellular Atlas of the Marmoset Brain,1U01MH114819-01,National Institute of Mental Health,Massachusetts Institute of Technology,9415538
 U01 Fischl grant,Imaging and analysis Techniques to Construct a Cell Census Atlas of the Human Brain,1U01MH117023-01,National Institute of Mental Health,Massachusetts General Hospital,9584061
 U01 Kriegstein grant,A Cellular Resolution Census of the Developing Human Brain,1U01MH114825-01,National Institute of Mental Health,University of California at San Francisco,9415877
 U01 Lein grant,A Multimodal Atlas of Human Brain Cell Types,1U01MH114812-01,National Institute of Mental Health,Allen Institute for Brain Science,9414852
 U01 Osten grant,Collaboratory for Atlasing Cell Type Anatomy in the Female and Male Mouse Brain,1U01MH114824-01,National Institute of Mental Health,Cold Spring Harbor Laboratory,9415873
-U01 Snyder-Mackler grant,Single cell transcriptional and epigenomic atlas of the macaque brain across the lifespan,1U01MH121260-01,National Institute of Mental Health,University of Washington,9852073
```

Comment:
keep U01 Snyder-Mackler

Reply:
Added BICCN-related grants and subprograms that don't have associated projects yet.